### PR TITLE
btpd: new package at 0.16

### PR DESCRIPTION
**Pull request by eudaldgr**
## Description of package

[The BitTorrent Protocol Daemon](https://github.com/btpd/btpd)

## Installed manifest of package

```
/var/db/kiss/installed/btpd/version
/var/db/kiss/installed/btpd/sources
/var/db/kiss/installed/btpd/manifest
/var/db/kiss/installed/btpd/depends
/var/db/kiss/installed/btpd/checksums
/var/db/kiss/installed/btpd/build
/var/db/kiss/installed/btpd/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/btpd.1
/usr/share/man/man1/btinfo.1
/usr/share/man/man1/btcli.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/bin/btpd
/usr/bin/btinfo
/usr/bin/btcli
/usr/bin/
/usr/

```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/btpd/build [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+autoreconf --verbose --install --force
```
Comment (dylanaraps) [inline community/btpd/build]: We don't run `autoreconf` unless necessary. Does it work when this is omitted?

File: community/btpd/build [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+./configure
```
Comment (dylanaraps) [inline community/btpd/build]: Missing `--prefix=/usr`.

File: community/btpd/build [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+make
+make PREFIX=/usr DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/btpd/build]: Remove `PREFIX=/usr`

File: community/btpd/depends [hunk 4]
```diff
@@ -0,0 +1,2 @@
+autoconf
+automake
```
Comment (dylanaraps) [inline community/btpd/depends]: Remove these if it works without `autoreconf`.

File: community/btpd/build (hunk 1 above)
Comment (dylanaraps) [reply community/btpd/build]: Wow. This project is really dead. Guess we need to keep this. Change it to: `autoreconf -fis`

Comment (eudaldgr) [reply community/btpd/build]: Don't work without autoreconf.

Comment (dylanaraps) [reply community/btpd/build]: Yup. Change it to `autoreconf -fis`.

File: community/btpd/build (hunk 3 above)
Comment (eudaldgr) [reply community/btpd/build]: Removed.

File: community/btpd/build (hunk 2 above)
Comment (eudaldgr) [reply community/btpd/build]: Added.

File: community/btpd/build (hunk 1 above)
Comment (eudaldgr) [reply community/btpd/build]: Changed.

File: community/btpd/build [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 autoreconf --verbose --install --force
 
-./configure
+./configure --prefix=/usr
```
Comment (dylanaraps) [inline community/btpd/build]:
```sh
./configure \
    --prefix=/usr
```

I suggest reading: https://k1ss.org/style

**Comment by dylanaraps** [comment]
Also, update the manifest list in the PR to reflect the changes in the build. :+1:

File: community/btpd/build (hunk 5 above)
Comment (eudaldgr) [reply community/btpd/build]: The rush.

**Comment by dylanaraps** [comment]
Thanks!

**Pull request merged**